2021-11-24 22:36:54,882 INFO    MainThread:43875 [wandb_setup.py:_flush():71] setting env: {}
2021-11-24 22:36:54,882 INFO    MainThread:43875 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-24 22:36:54,882 INFO    MainThread:43875 [wandb_init.py:_log_setup():371] Logging user logs to /home/snamburi/Debarshi/wandb/run-20211124_223654-3ci0b9dt/logs/debug.log
2021-11-24 22:36:54,883 INFO    MainThread:43875 [wandb_init.py:_log_setup():372] Logging internal logs to /home/snamburi/Debarshi/wandb/run-20211124_223654-3ci0b9dt/logs/debug-internal.log
2021-11-24 22:36:54,883 INFO    MainThread:43875 [wandb_init.py:init():404] calling init triggers
2021-11-24 22:36:54,883 INFO    MainThread:43875 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-24 22:36:54,883 INFO    MainThread:43875 [wandb_init.py:init():449] starting backend
2021-11-24 22:36:54,883 INFO    MainThread:43875 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-24 22:36:54,916 INFO    MainThread:43875 [backend.py:ensure_launched():199] starting backend process...
2021-11-24 22:36:54,939 INFO    MainThread:43875 [backend.py:ensure_launched():204] started backend process with pid: 43888
2021-11-24 22:36:54,940 INFO    MainThread:43875 [wandb_init.py:init():458] backend started and connected
2021-11-24 22:36:54,941 INFO    MainThread:43875 [wandb_init.py:init():519] updated telemetry
2021-11-24 22:36:54,945 INFO    MainThread:43875 [wandb_init.py:init():549] communicating current version
2021-11-24 22:36:55,794 INFO    MainThread:43875 [wandb_init.py:init():554] got version response 
2021-11-24 22:36:55,794 INFO    MainThread:43875 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-24 22:36:56,007 INFO    MainThread:43875 [wandb_init.py:init():592] starting run threads in backend
2021-11-24 22:36:57,526 INFO    MainThread:43875 [wandb_run.py:_console_start():1816] atexit reg
2021-11-24 22:36:57,527 INFO    MainThread:43875 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-24 22:36:57,528 INFO    MainThread:43875 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-24 22:36:57,530 INFO    MainThread:43875 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-24 22:36:57,530 INFO    MainThread:43875 [wandb_init.py:init():619] run started, returning control to user process
2021-11-24 22:36:57,530 INFO    MainThread:43875 [wandb_run.py:_config_callback():962] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-11-24 22:36:58,500 INFO    MainThread:43875 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-24 22:36:58,500 INFO    MainThread:43875 [wandb_run.py:_restore():1758] restore
2021-11-24 22:37:00,674 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:37:00,877 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 22:37:01,051 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 48973
}

2021-11-24 22:37:01,174 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 48973
}

2021-11-24 22:37:01,277 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}

2021-11-24 22:37:01,380 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}

2021-11-24 22:37:01,483 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}

2021-11-24 22:37:01,586 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}

2021-11-24 22:37:01,689 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}

2021-11-24 22:37:01,838 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}

2021-11-24 22:37:02,100 INFO    MainThread:43875 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 48973
  total_bytes: 48973
}
local_info {
}

2021-11-24 22:37:03,273 INFO    MainThread:43875 [wandb_run.py:_append_files():2184] logging synced files
